### DISCUSSION TITLE
can a base license only be used on one development server?

### DISCUSSION THREAD

**Discussion by hanswang73** [dashingsoft/pyarmor#1892] [Q&A]
May I ask, can a base license only be used on one development server? Is it possible to migrate it to other development servers for use?

**Comment by jondy** [comment]
Run `pyarmor reg xxx.zip` in another machine

**Reply by hanswang73** [reply]
I tried:

INFO     Python 3.10.14
INFO     Pyarmor 8.5.0 (trial), 000000, non-profits
INFO     Platform linux.x86_64
INFO     register "pyarmor-regfile-4564.zip"
INFO     update license token
ERROR    request license token failed (101): Success
ERROR    somthing is wrong
*===========================================================*
*  Please check                                             *
*    https://pyarmor.readthedocs.io/en/latest/questions.html*
*  or run `pyarmor man` to find solutions quickly           *
*                                                           *
*  It's recommand to report issue by `pyarmor man` in order *
*  to provide necessary information, and avoid dupcliated   *
*  issues or unclear question.                              *
*===========================================================*

**Comment by jondy** [comment]
Make sure this machine is online, also take this check list for registration issue
https://pyarmor.readthedocs.io/en/latest/reference/solutions.html

**Reply by hanswang73** [reply]
my license is basic, do every step in https://pyarmor.readthedocs.io/en/latest/reference/solutions.html, can not find the reason yet.

**Reply by hanswang73** [reply]
my reg file is pyarmor-regfile-4564.zip

**Reply by hanswang73** [reply]
root@autodl-container-700945b505-d1ec0e4d:~/autodl-fs/_HansCode/ChatInfoDev# pyarmor reg pyarmor-regfile-4564.zip

INFO     Python 3.10.15
INFO     Pyarmor 8.5.0 (trial), 000000, non-profits
INFO     Platform linux.x86_64
INFO     register "pyarmor-regfile-4564.zip"
INFO     update license token
ERROR    request license token failed (101): Success
ERROR    somthing is wrong
*===========================================================*
*  Please check                                             *
*    https://pyarmor.readthedocs.io/en/latest/questions.html*
*  or run `pyarmor man` to find solutions quickly           *
*                                                           *
*  It's recommand to report issue by `pyarmor man` in order *
*  to provide necessary information, and avoid dupcliated   *
*  issues or unclear question.                              *
*===========================================================*

**Reply by hanswang73** [reply]
upgrade from 8.5.0 to 8.5.2, still error
upgrade to 8.5.12, re-register SUCCESS .
